R/check_package.R

```diff
@@ -1,4 +1,22 @@
-#' A standardised test for R packages
+#' Run the complete set of standardised tests on a package
+#'
+#' A convenience function that runs all packages related tests in sequence.
+#' The details lists the relevant functions.
+#' When you fixed a problem, you can speed things up by running only the related
+#' check.
```

Comment:
Is there an easy way to find out which check had failed?  If this is not fully clear from the check report, it could be useful to explain it shortly here.